### ISSUE TITLE
create DPOR whitepaper (how does Gridcoin weight blocks)

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#3]
There is 
* an existing doc available, see hangout #1: 0:15:47 and 
* also a summary (with graphs) https://steemit.com/gridcoin/@cm-steem/pos-dpor-stake-weight-balance

bounty is also possible

**Comment by grctest** [comment]
I agree that a bounty could be worthwhile, the creation of a DPOR whitepaper will ensure that we can all understand the current state of DPOR & then improve upon it.

**Comment by Quezacoatl1** [comment]
I'll add 10k GRC as a bounty to this issue, what is the current complete bounty @Erkan-Yilmaz ?
I believe a whitepaper is necessary.

@tomasbrod would you feel comfortable in (co-)writing one? I would/could add some content.

I found some google docs to start with: https://docs.google.com/document/d/1PUmLpL3jb-Dhf9-60MPpCYpCvBukUIWeKO8Pww0bhII

Well, it might be outdated.

**Comment by gregagnew** [comment]
Is there any progress on this?

**Comment by Quezacoatl1** [comment]
@tomasbrod you "liked" my post, was this similar to "ill do it"? :D

**Comment by gregagnew** [comment]
Looking further into this, perhaps we should wait until a new DPoR block is developed? It seems like that is our top priority at the moment.

**Comment by tomasbrod** [comment]
I am not interested in writing a white paper, it is too much work for me. I will however, review when something is written, give advice on how something works, help analyze crypto and write short explanations of how something works.
So you @Quezacoatl1 can count on my help, but I won't actually do it :)
@gregagnew Top priority currently is chain security with v8 stake kernel.

**Comment by gregagnew** [comment]
I can write this. I'd prefer to write for proposed solution to the DPoR, that I assume will accompany removing gridcoin team requirements (issue #94). Can we establish a pseudo-code mechanism for this?

In the mean time I will fill in the other parts.

Is there a media kit for gridcoin, including logos, motto's, palettes and fonts?

**Comment by hovai** [comment]
I will commit 5000 GRC to the whitepaper bounty if there is a described structure which approves a whitepaper, be it a vote in the client or whatever.

**Comment by Erkan-Yilmaz** [comment]
see also here:
* https://github.com/gridcoin-community/Whitepaper

**Comment by Quezacoatl1** [comment]
Sent 10k to S66MFdorkAc8oSt7ieW95oWGnLQodzZFkn

https://gridcoinstats.eu/block/1232471/3e70d60381ba9342c7b88727e873c90082afe789ee548ad199f768ad813fd9a9
